### ISSUE TITLE
The question about the signs in the formula: loss = log_p_x_given_z + log_p_z_given_c - log_p_c +  log_q_c_given_x - log_q_z_given_x   

### ISSUE THREAD

**Issue by JunetaeKim** [mperezcarrasco/Pytorch-VaDE#3]
"loss = log_p_x_given_z + log_p_z_given_c - log_p_c + log_q_c_given_x - log_q_z_given_x

Regarding the sign, shouldn't it be + log_q_z_given_x instead of - log_q_z_given_x?"
